### PR TITLE
Disallow flv files from being uploaded

### LINKED ISSUE #435: Block FLV uploads
Flash isn't allowed on VIP. We need to block `.flv` files from being uploaded.

### PR DESCRIPTION
**Pull request by joshbetz**
Fixes #435

### CODE REVIEW

File: misc.php [hunk 1]
```diff
@@ -7,6 +7,11 @@
 License: GPL version 2 or later - http://www.gnu.org/licenses/old-licenses/gpl-2.0.html
 */
 
+add_filter( 'upload_mimes', function( $mimes ) {
+	unset( $mimes['flv'] );
+	return $mimes;
+});
```
Comment (nickdaugherty) [inline misc.php]: Should we set the priority really high, so that other client code can't come along and re-enable it?

Comment (joshbetz) [inline misc.php]: I guess that wouldn't hurt, but obviously malicious code could just set the priority higher than we do.

Comment (joshbetz) [inline misc.php]: Or we could set the priority to `PHP_INT_MAX` ;)

Comment (nickdaugherty) [inline misc.php]: Yeah they could always 1-up us - though us setting a high priority is also a signal to devs that this is meant to be the final say and not overridden, and makes an override more deliberate and perhaps easier to detect.

**Comment by nickdaugherty** [comment]
🎉

**Comment by joshbetz** [comment]
r87840-deploy

**Pull request merged**